.github/release.yml

```diff
@@ -0,0 +1,97 @@
+name: kong-wallet-jwt Binary Release
+
+on:
+  push:
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+"     # Push events to matching v*, i.e. v1.0, v20.15.10
+      - "v[0-9]+.[0-9]+.[0-9]+-rc*" # Push events to matching v*, i.e. v1.0-rc1, v20.15.10-rc5
+
+jobs:
+  build_init:
+    runs-on: ubuntu-latest
+    name: Build Init
+    steps:
+      - id: version
+        run: echo "::set-output name=version::${GITHUB_REF##*/}"
+    outputs:
+      version: ${{ steps.version.outputs.version }}
+      go_version: "1.17"
+
+  build_osx:
+    runs-on: macos-latest
+    needs:
+      - build_init
+    name: Build OSX
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Install osx tools
+        run: |
+          brew tap mitchellh/gon
+          brew install mitchellh/gon/gon
+      - name: Setup go
+        uses: actions/setup-go@v2.1.4
+        with:
+          go-version: ${{ needs.build_init.outputs.go_version }}
+      - name: Import Code-Signing Certificates
+        uses: Apple-Actions/import-codesign-certs@v1
+        with:
+          p12-file-base64: ${{ secrets.CODESIGNING_P12_BASE64 }}
+          p12-password: ${{ secrets.CODESIGNING_P12_PASSWORD }}
+      - name: Build osx binary
+        run: |
+          make bin
+      - name: Sign the mac binaries with Gon
+        env:
+          AC_USERNAME: ${{ secrets.AC_USERNAME }}
+          AC_PASSWORD: ${{ secrets.AC_PASSWORD }}
+        run: |
+          gon -log-level=info -log-json ./gon.json
+      - uses: actions/upload-artifact@v2
+        with:
+          name: osx-zip
+          path: build/*.zip
+
+  build_linux:
+    runs-on: ubuntu-latest
+    needs:
+      - build_init
+    name: Build Linux
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup go
+        uses: actions/setup-go@v2.1.4
+        with:
+          go-version: ${{ needs.build_init.outputs.go_version }}
+      - name: Build linux binary
+        run: |
+          make bin
+      - uses: actions/upload-artifact@v2
+        with:
+          name: linux-zip
+          path: build/*.zip
+
+  create_release:
+    runs-on: ubuntu-latest
+    needs:
+      - build_init
+      - build_osx
+      - build_linux
+    name: Create Release
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Create release ${{ needs.build_init.outputs.version }}
+        uses: actions/create-release@v1
+        id: create_release
```

Comment:
I believe you need to download the artifacts created and then upload them to the release. Similar to (this)[https://github.com/provenance-io/provenance/blob/main/.github/workflows/release.yml#L139-L158)

Reply:
Ah yeah that is missing, added both mac and linux uploads